### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by jamarshon**
Ran make html for gh-pages

### CODE REVIEW

**Comment by jamarshon** [comment]
@soumith

**Comment by cpuhrsch** [comment]
Do you need to redo this due to the recent merge?

**Comment by jamarshon** [comment]
I did it in commit df5d216 (different from 8e29244 which is the pre-merge)

File: _modules/torchaudio/datasets/vctk.html [hunk 1]
```diff
@@ -293,7 +296,7 @@ <h1>Source code for torchaudio.datasets.vctk</h1><div class="highlight"><pre>
 <span class="sd">        download (bool, optional): If true, downloads the dataset from the internet and</span>
 <span class="sd">            puts it in root directory. If dataset is already downloaded, it is not</span>
 <span class="sd">            downloaded again.</span>
-<span class="sd">        transform (callable, optional): A function/transform that  takes in an PIL image</span>
+<span class="sd">        transform (callable, optional): A function/transform that  takes in an raw audio</span>
```
Comment (cpuhrsch) [inline _modules/torchaudio/datasets/vctk.html]: "an raw audio" - probably need to fix that doc in a separate PR.

**Pull request merged**